data/MohdSaquib114

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Mohd Saquib Mansoori",
+  "bio": "Experienced fullstack developer with expertise in building end-to-end web applications using modern technologies and best practices.",
+  "avatar": "https://github.com/MohdSaquib114.png",
```

Comment:
The avatar tag is no longer required, consider taking a look at the [QuickStart Documentation](https://linkfree.io/docs/quickstart-json), if you haven't already for reference to this topic.

![av](https://github.com/EddieHubCommunity/LinkFree/assets/19821445/7718d5a9-93ca-49c1-b2c4-94602d7ca017)
```suggestion
```